src/components/ac-layer/ac-layer.component.ts

```diff
@@ -16,6 +16,8 @@ import { StaticPolylineDrawerService } from '../../services/static-polyline-draw
 import { PolygonDrawerService } from '../../services/polygon-drawer/polygon-drawer.service';
 import { ArcDrawerService } from '../../services/arc-drawer/arc-drawer.service';
 import { PointDrawerService } from '../../services/point-drawer/point-drawer.service';
+import 'rxjs/add/observable/merge';
```

Comment:
checkout `src/operators.ts` that i added recently , where operators are loaded globally